### ISSUE TITLE
Release v10.3.0 is no compatible with CentOS 8 and least

### ISSUE THREAD

**Issue by lainosantos** [heketi/heketi#1848]
The release 10.3.0 artifacts for heketi-cli is not compatible anymore with CentOS 8, because it's requiring glibc 2.32 and the SO provides 2.28. Any necessary requirements for glibc 2.32?

**Comment by phlogistonjohn** [comment]
I can confirm the issue.

It's a bit odd because I used the same build procedure for a while now, using the same basic container source we use for the "fromsource" builds. All I can figure is that this should have actually failed sooner, once Fedora 33 was released except that some caching on my end must have retained F32?

Regardless, we'll look into seeing what we can do.

**Comment by amgads** [comment]
Please see issue #1869 as well -- Does v10.3.0 intended for EL8 only? and if yes, CentOS 8 doesn 't have glibc 2.32, only 2.28.

**Comment by phlogistonjohn** [comment]
(Re-posting a version of what I wrote on issue #1869 - for wide as possible awareness)

As for the timeliness of fixes/releases/etc I'll refer you to our [readme](https://github.com/heketi/heketi/blob/master/README.md) which currently says:
```
As of Heketi release v10.2 the project is considered to be in maintenance only status. This means that only major bugs, performance regressions, or security defects are being considered for inclusion by the project team. Please also expect slow replies to issues.
```

That said, in order to continue to maintain heketi we're looking at switching to go modules in order to support newer version of golang in order to continue to take security fixes in Go. Thus we do expect to do another release this year - as part of the release process we will take the glibc ABI issue into account and make an effort to build heketi such that the binary releases work on a wider variety of base distributions.

**Comment by phlogistonjohn** [comment]
This should be fixed in the builds provided by heketi v10.4.0. We now build the binaries attached to the release on CentOS7 to ensure the use of older glibc abi.
